apps/laboratory/tests/verify.spec.ts

```diff
@@ -184,15 +182,13 @@ testMEthersVerifyDomainMismatch(
 
     const uri = await modalPage.getConnectUri()
     await walletPage.connectWithUri(uri)
-    await expect(walletPage.page.getByText('Invalid Domain')).toBeVisible()
     await walletPage.handleSessionProposal(DEFAULT_SESSION_PARAMS)
     await modalValidator.expectConnected()
     await walletValidator.expectConnected()
 
     await modalPage.sign()
     const chainName = DEFAULT_CHAIN_NAME
     await walletValidator.expectReceivedSign({ chainName })
-    await expect(walletPage.page.getByText('Invalid Domain')).toBeVisible()
```

Comment:
why remove this?

Reply:
in this version Sign SDK automatically gets domain url from the page regardless of what is set as metadata.url